tests/orbit/utils/params_tuning.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+
+from orbit.models import DLT, LGT
+from orbit.utils.params_tuning import grid_search_orbit
+
+
+@pytest.mark.parametrize(
```

Comment:
it is better to include the criteria (`bic` or `backtest`) as a parameter for unit test.